benchmark/doublecoset/doublecoset1.tst

```diff
@@ -0,0 +1,18 @@
+#############################################################################
+##
+#W  doublecoset.tst                                     Alexander Hulpke
+##
+##
+#Y  Copyright (C)  1997,  Lehrstuhl D für Mathematik,  RWTH Aachen,  Germany
+##
+##  This  file  tests for double coset calculations
+##
+gap> START_TEST("doublecoset.tst");
+gap> g:=SimpleGroup("J2");;
+gap> m:=MaximalSubgroupClassReps(g);;
+gap> u:=First(m,x->Index(g,x)=10080);;
```

Comment:
This isn't really a benchmark for double cosets, as the index is small enough to even calculate orbits on cosets. (In fact the whole calculation on a group of order barely 600k is a very easy case)
The issue that the new code addresses is with subgroups of large index that do not have satisfactory intermediate subgroups (which is a given if they are maximal), so that a Leiterspiel with up-step is required.

One could try to have larger examples without Leiterspiel (just requiring to find intermediate subgroups), but as long as indices are not in the millions this is all childs play.